modules/hp_graph/templates/paragraph--hp-graph-field.html.twig

```diff
@@ -0,0 +1,18 @@
+{#
+/**
+ * @file
+ * HP Graph Field paragraph.
+ *
+ * @ingroup themeable
+ */
+#}
+
+<section>
+	<div class="hp_graph_container">
+		<span class="percent-label">{{-content.field_hp_graph_label[0]-}}-{{-content.field_hp_graph_percent[0]-}}%</span>
```

Comment:
are the spaceless ({{- and -}}) needed? It would make for a lot cleaner code if not present. Note that best practice would be to use a space, ie {{ myvar }} as opposed to {{myvar}} for readability and drupal code standards. https://www.drupal.org/docs/develop/coding-standards/twig-coding-standards#whitespace